conjure_oxide/src/utils/testing.rs

```diff
@@ -220,5 +221,40 @@ pub fn read_rule_trace(
         writeln!(file, "{}", rules_trace.join("\n"))?;
     }
 
+    if accept {
+        std::fs::copy(
+            format!("{path}/{test_name}-generated-rule-trace.json"),
+            format!("{path}/{test_name}-expected-rule-trace.json"),
+        )?;
+    }
+
     Ok(rules_trace)
 }
+
+// use std::fs::{self};
```

Comment:
Should we delete this commented out function, or we are going to use it somewhere?